listeners.go

```diff
@@ -289,8 +289,10 @@ func (na NetworkAddress) String() string {
 	return JoinNetworkAddress(na.Network, na.Host, na.port())
 }
 
-func isUnixNetwork(netw string) bool {
-	return netw == "unix" || netw == "unixgram" || netw == "unixpacket"
+// IsUnixNetwork returns true if the netw is a unix network.
+func IsUnixNetwork(netw string) bool {
```

Comment:
Renaming this to be exported doesn't work because this exact name is already used in another file. See the linter errors.

Reply:
Argh I forgot to rename the unix-only usages.